pom.xml

```diff
@@ -7,6 +7,26 @@
     <groupId>org.example</groupId>
     <artifactId>TwittersAPiNew</artifactId>
     <version>1.0-SNAPSHOT</version>
```

Comment:
Please follow the maven naming conventions for artifactId: https://maven.apache.org/guides/mini/guide-naming-conventions.html